homeassistant/components/overseerr/coordinator.py

```diff
@@ -0,0 +1,38 @@
+"""Overseer coordinator s."""
+from datetime import timedelta
+import logging
+
+from overseerr_api import ApiClient, Configuration, RequestApi
+from overseerr_api.exceptions import OpenApiException
+from overseerr_api.models import RequestCountGet200Response
+from urllib3.exceptions import MaxRetryError
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class OverseerrUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class OverseerrUpdateCoordinator(DataUpdateCoordinator[RequestCountGet200Response]):
```
coordinator should by typed with data type